molecule/default/roles/test_glauth/tasks/verify.yml

```diff
@@ -0,0 +1,58 @@
+---
+- name: Detect glauth.toml file
+  ansible.builtin.stat:
+    path: /etc/glauth/glauth.toml
+  register: glauth_toml
+
+- name: Check syntax of glauth config
+  ansible.builtin.command:
+    cmd: /usr/bin/glauth -c {{ glauth_toml.stat.path }} --check-config
+  register: check_config
+  failed_when: false
+  changed_when: check_config.rc != 0
```

Comment:
Hmm, I did it this way because I was only collecting info, to assert later. But the collection can be its own assertion I guess (and these lines removed completely).

Done.